cmd/launcher/extension.go

```diff
@@ -139,31 +147,8 @@ func createExtensionRuntime(ctx context.Context, k types.Knapsack, launcherClien
 					}
 
 					// The runner allows querying the osqueryd instance from the extension.
-					// Used by the Enroll method below to get initial enrollment details.
 					ext.SetQuerier(runner)
 
-					// It's not clear to me _why_ the extension
-					// ever called Enroll here. From what
-					// I can tell, this would cause the
-					// launcher extension to do a bunch of
-					// initial setup (create node key,
-					// etc). But, this is also done by
-					// osquery. And having them both
-					// attempt it is a bit racey.  I'm not
-					// so confident to outright remove it,
-					// so it's gated behind this debugging
-					// environment.
```

Comment:
hahahahahaha. So we did this, I didn't understand, and you cleaned it up. Beautiful